kml.cc

```diff
@@ -792,33 +792,33 @@ void kml_output_trkdescription(const route_head* header, const computed_trkdata*
   }
   const char* distance_units;
   double distance = fmt_distance(td->distance_meters, &distance_units);
-  kml_td(hwriter, QStringLiteral("Distance"), QStringLiteral(" %1 %2 ").arg(QString::number(distance, 'f', 1)).arg(distance_units));
+  kml_td(hwriter, QStringLiteral("Distance"), QStringLiteral(" %1 %2 ").arg(QString::number(distance, 'f', 1), distance_units));
```

Comment:
The original guidance for localizataion was to have user-visible generated strings as .arg so that translators could reorder them if needed. We've generally made such a mess of that this is probably not helpful now.